server/spec/candlepin_scenarios.rb

```diff
@@ -486,13 +605,8 @@ def create_candlepin_export_update
     @cp.add_content_to_product(@owner['key'], product2.id, arch_content.id)
 
     end_date = Date.new(2025, 5, 29)
-    @cp.create_subscription(@owner['key'], product1.id, 12, [], '', '12345', '6789', nil, end_date)
-    @cp.create_subscription(@owner['key'], product2.id, 14, [], '', '12345', '6789', nil, end_date)
-
-    @cp.refresh_pools(@owner['key'])
-
-    pool1 = @cp.list_pools(:owner => @owner.id, :product => product1.id)[0]
-    pool2 = @cp.list_pools(:owner => @owner.id, :product => product2.id)[0]
+    pool1 = create_pool_and_subscription(@owner['key'], product1.id, 12, [], '', '12345', '6789', nil, end_date)
+    pool2 = create_pool_and_subscription(@owner['key'], product2.id, 14, [], '', '12345', '6789', nil, end_date)
```

Comment:
not in this case. if we use the skip flag , we do not return the pool reference.